docs/wallet/web-wallet.md

```diff
@@ -201,26 +235,103 @@ In the last step, you will need to check your authenticator tool to retrieve the
 
 ### **Unguarding your account**
 
-Unguarding your account consists of sending an ```UnGuardAccount``` transaction to your own address. You can do it by simply using the **Unguard Account** button from the Guardian section:
+Unguarding your account consists of sending a guarded ```UnGuardAccount``` transaction to your own address. You can do it by simply using the **Unguard Account** button from the Guardian section:
 
-![img](/wallet/web-wallet/guardian_unguard.png)
+![img](/wallet/web-wallet/guardian_step44.png)
 
 And confirming the transaction after checking the validity of it:
 
 ![img](/wallet/web-wallet/guardian_unguard2.png)
 
+And validating it via the authenticator app:
```

Comment:
before this section end, please also explain the Mainnet duration for this operation

Reply:
good point